packages/web/app/components/Field/DateField.jsx

```diff
@@ -81,7 +81,7 @@ export const DateInput = ({
 
   const onValueChange = useCallback(
     event => {
-      if (event.target.validity.badInput) {
+      if (event.target.validity?.badInput) {
```

Comment:
`validity` is not a field on the target when the arrows are clicked (and nor do we require it: the value from an arrow click is always valid)